docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -314,6 +314,25 @@ xpack.ml.model_repository: file://${path.home}/config/models/`
 master-eligible nodes one by one.
 
 
+[discrete]
+== Testing ELSER
+
+You can test the deployed model in {kib}. Navigate to **Model Management** > 
+**Trained Models**, locate the deployed ELSER model in the list of trained 
+models, then select **Test model** from the Actions menu.
+
+You can use data from an existing index to test the model. Select the index, 
+then a field of the index you want to test ELSER on. Provide a search query and 
+click **Test**. Evaluating model recall is simpler when using a query related to 
+the documents.
+
+The results contain the search relevance score (higher means more relevant) and 
+the respective document for each hit.
```

Comment:
Maybe something like this would be clearer?
```
The results contain a list ten random values for the selected field along with a score showing how relevant each doc is to the user inputted query. The higher the score, the more relevant the doc is.
```